### PR TITLE
Feature/18 default config and bootstrapping

### LINKED ISSUE #18: Add default config and bootstrap

### PR DESCRIPTION
**Pull request by brettminnie**
Will close #18

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Looks good so far :)

**Comment by brettminnie** [comment]
@eddiejaoude think this is ready for you will close #18

File: config/exclusions/exclusions.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Example exclusions
+#
+#exclusions:
+#  BDBStudios\BreakfastSerializerTest\Fixtures\ExclusionClass:
+#    excludeVariables:
+#      - internalProperty
+#      - isExcluded
```
Comment (eddiejaoude) [inline config/exclusions/exclusions.yml]: Meant to be commented out?

Comment (brettminnie) [inline config/exclusions/exclusions.yml]: yes, hence the Example exclusions at the top :)

Comment (eddiejaoude) [inline config/exclusions/exclusions.yml]: Ah ok :)

**Pull request merged**